### ISSUE TITLE
v0.7.3 does not include #16

### ISSUE THREAD

**Issue by aiotter** [atomvm/atomvm_packbeam#19]
It seems v0.7.3 is for #16 (65e9053a99912c9550e639d1e7f299d01ef6448c), however  65e9053a99912c9550e639d1e7f299d01ef6448c is not on the history of 0.7.3 (81543e5fcb80a1a7431d1f5fa3798eb4096ea265).

**Comment by UncleGrumpy** [comment]
Indeed. I will have to investigate why there are missing commits in the 0.7.3 tag, in the master branch the tag is clearly made after #16 was merged. At this point I do not see why the commit is not included up when checking out the 0.7.3 tag.

**Comment by UncleGrumpy** [comment]
We really need to make another release in the near future, we will have to make sure that it includes all of the missing commits, and likely amend the CHANGELOG to reflect the release that the changes are actually included in.

Thank you so much for pointing this out, it has gone unnoticed for quite some time. ;-)

**Comment by karlsson** [comment]
It would be nice if you publish 0.7.4 on hex.pm too, once it is released. Seems 0.7.3 was not published.

**Comment by UncleGrumpy** [comment]
That is absolutely the plan! We had a temporary problem with not being able to release on hex.pm, but that is sorted out now. I did not release 0.7.3 to hex.pm late, because somehow that was a bad tag, and did not actually include the fix that it was intended to address. I did not want to try retagging the release on GitHub, since this could cause problems for users who had pulled the tag and had it cached.

I did discover a few housekeeping chores that I need to take care of before making this release.

**Comment by karlsson** [comment]
Thanks @UncleGrumpy ,
The 0.7.2 release works fine for me at the moment, so take your time :smile: .
